src/Altinn.Platform/Altinn.Platform.PDF/src/main/java/altinn/platform/pdf/models/FormLayoutElement.java

```diff
@@ -14,6 +14,7 @@
   private String id;
   private HashMap<String, String> dataModelBindings;
   private TextResourceBindings textResourceBindings;
+  private String optionsId;
   private boolean simplified;
   private List<String> children;
   private int count;
```

Comment:
We probably also should consider some Inheritance on this FormLayoutElement class. Make subclasses for props that are specific to each component. Maybe that is work for some other time.

Reply:
Sounds like a job for a Java developer, @tba76  :P